email_templates/scratch_org_expiry_email.txt

```diff
@@ -1,9 +1,13 @@
 {% load i18n %}{% trans "Expiry Alert from Metecho" %}
 
 {% blocktrans trimmed count counter=days %}
-Your Dev Org for "{{ task_name }}" on {{ project_name }} epic "{{ epic_name }}" is set to expire in one day on {{ expiry_date }} You have uncommitted changes on your org that will be deleted. If you do not want to lose these changes, please log into Metecho as user "{{ user_name }}", navigate to the task, and click the "Retrieve Changes from Dev Org" button. We appreciate your contribution to the Salesforce community. Thanks!
+Your Dev Org for {{ task_name }} is set to expire in one day on {{ expiry_date }}.
```

Comment:
I don't fully understand the `blocktrans` changes here, but I assume it's 🔥 😂

Reply:
There's two things going on:

1. Use `{{ task_name }}` without conditionals since `Task.full_name` will handle mentioning the epic or not (as you noted)
2. "Factor out" the repeating part of the message ("You have uncommitted changes...") so translators only need to focus on the day/days difference of the first sentence.

That being said we aren't currently generating the string catalog for Django so we'll know how it works later 😅